### ISSUE TITLE
[Rule] Duplicate argument name

### ISSUE THREAD

**Issue by bhirsz** [MarketSquare/robotframework-robocop#401]
Duplicating argument name should be detected. In this example:
```
*** Test Cases ***
Test keyword with duplicate arg
    Keyword    1    2


*** Keywords ***
Keyword
    [Arguments]    ${var}    ${var}
    Log  ${var}
```
Robocop doesn't warn about duplicate argument. When running the code Robot Framework also doesn't complain and just overwrite first ${var} with second one:
![image](https://user-images.githubusercontent.com/8532066/127778119-5f4d4166-d448-4aff-9b0d-746268adeb21.png)
